### PR TITLE
St igor budnitski lesson2 level8

### PR DESCRIPTION
**Pull request by igor-budnitski**
Lesson2 - level8 -homework
Smart calculator, where user can enter numbers and select required operation

### CODE REVIEW

File: src/main/java/students/student_igor_budnitski/HelloWorld.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package students.student_igor_budnitski;
 
+import java.util.Scanner;
```
Comment (antdolg1) [inline src/main/java/students/student_igor_budnitski/HelloWorld.java:3]: Unused import. Please remove.

File: src/main/java/students/student_igor_budnitski/level8/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package students.student_igor_budnitski.level8;
+
+
```
Comment (antdolg1) [inline src/main/java/students/student_igor_budnitski/level8/Calculator.java:3]: Redundant empty line.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Changes needed.

File: src/main/java/students/student_igor_budnitski/level8/Calculator.java (hunk 2 above)
Comment (igor-budnitski) [reply src/main/java/students/student_igor_budnitski/level8/Calculator.java]: Changes were applied and empty line was removed.

**Review by antdolg1 — APPROVED** [review]

**Pull request merged**